_posts/2024-01-02-resource-recommendations.html

```diff
@@ -0,0 +1,200 @@
+---
+author:
+duration: 4 Min Read
```

Comment:
Should be "Upstream"
![Screenshot 2024-04-17 at 9 42 11 AM](https://github.com/UpstreamCode/upstreamcode.github.io/assets/31777263/9a9a03fe-7cc2-4c31-8fcc-8548ea6aa810)